common/utils/check_cstor_pool_pod.yml

```diff
@@ -0,0 +1,7 @@
+---
+   - name: Get the cstor pool pods which remains in a pending state
+     shell: > 
+         kubectl get po -n {{ pool_namespace }} -l {{ pool_label }} 
+         -o jsonpath='{.items[?(@.status.phase=="Pending")].metadata.name}' | wc -w
+     register: pod_pending
+     failed_when: "pod_pending.stdout != 0"
```

Comment:
can take this now, but really recommend to check if this is a subset of other utils. Lets try to avoid redundancy in utils.